web/src/helpers/currencyFormatter.js

```diff
@@ -1,4 +1,8 @@
 export default function currencyFormatter(value, currency, locale) {
+  if (!value && value !== 0) {
+    return '';
```

Comment:
```suggestion
    return null;
```

se o retorno for `null` não vai passar no v-if

Reply:
Vai funcionar igual, vamos manter